_sass/main.scss

```diff
@@ -442,14 +442,14 @@ table code {
   color: $yellow;
 }
 
-.blog-page, .post-page, .getting-started-page, .documentation, .OSD-2022-page, .hacktoberfest-page, .contribute-page, .guides-page, .setup-elytron {
+.blog-page, .post-page, .getting-started-page, .documentation, .OSD-2023-page, .hacktoberfest-page, .guides-page, .setup-elytron {
```

Comment:
I think we can keep the .contribute-page here since it will be needed when we switch back.

Reply:
Same with .hacktoberfest-page.

Reply:
fixed!